Dockerfile

```diff
@@ -45,14 +45,9 @@ RUN mkdir -p /data
 ENV NUA_ZONE=europe-1
 ENV NUA_API_KEY=
 ENV NUCLIA_PUBLIC_URL=https://{zone}.nuclia.cloud
-
+ENV PYTHONUNBUFFERED=1
```

Comment:
I've usually used this in the past because it's possible that you lose logs in a crash.

I think with the amount of logs we have, we should be okay!